chassis/core/pom.xml

```diff
@@ -276,5 +276,11 @@
             <artifactId>jackson-core</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
Don't we have JUnit's dependency already somewhere? At least to get version number.